Input/tests/InputField-specs.js

```diff
@@ -1,58 +1,68 @@
-import {mount} from 'enzyme';
-import {InputField} from '../';
 import Spotlight from '@enact/spotlight';
+import '@testing-library/jest-dom';
+import {fireEvent, render} from '@testing-library/react';
+
+import {InputField} from '../';
 
 const isPaused = () => Spotlight.isPaused() ? 'paused' : 'not paused';
 
 describe('InputField Specs', () => {
 	test('should have an input element', () => {
-		const subject = mount(
-			<InputField />
+		const {getByTestId} = render(
+			<InputField data-testid="inputField" />
 		);
 
-		expect(subject.find('input')).toHaveLength(1);
+		const inputField = getByTestId('inputField');
+		const expected = 'inputField';
+		const actual = inputField.className;
+
+		expect(actual).toContain(expected);
 	});
```

Comment:
You can simply check if input is in the document, and not look for className. What do you think?
```suggestion
	test('should have an input element', () => {
		const {getByTestId} = render(
			<InputField data-testid="inputField" />
		);

		const inputField = getByTestId('inputField');

		expect(inputField).toBeInTheDocument();
	});
	
```

Reply:
Done